### PR TITLE
Implement new features, update readme

### PR DESCRIPTION
**Pull request by ClintonYounge**
**Added Features**

- Feature to delete all checked items.
- Feature to allow the dragging of tasks up and down the list.

### CODE REVIEW

**Comment by ClintonYounge** [comment]
Hello code reviewer. I accidentally submitted the additional tasks for the project. 

"Feature to allow the dragging of tasks up and down the list" is an additional feature that I should've done in a separate pull request.

File: src/module/tasklist.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const taskList = document.querySelector('#task-list');
-const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
 // Save tasks to local storage
 function saveTasks() {
```
Comment (OLIPLICHE) [inline src/module/tasklist.js:5]: 👍 👍 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @ClintonYounge, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights
- The updated tasks list is stored in local storage👏 👏
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/module/tasklist.js (hunk 1 above)
Comment (ClintonYounge) [reply src/module/tasklist.js:5]: You're a great reviewer. I saw this but I didn't change it. Wanted to see if you would recommend hehe

**Pull request merged**